site/en/tutorials/text/word2vec.ipynb

```diff
@@ -269,7 +269,7 @@
       },
       "outputs": [],
       "source": [
-        "# Load the TensorBoard notebook extension\n",
```

Comment:
The original was correct IMHO ("the... notebook extension")

Reply:
Thanks for the Fix!!!